### PR TITLE
add norm to fpn's conv

### PR DESCRIPTION
**Pull request by Noplz**
add norm to fpn's conv
ref：https://github.com/facebookresearch/Detectron/blob/master/detectron/modeling/FPN.py

### CODE REVIEW

File: PaddleCV/PaddleDetection/ppdet/modeling/backbones/fpn.py [hunk 1]
```diff
@@ -39,35 +40,49 @@ class FPN(object):
...
     """
 
     def __init__(self,
                  num_chan=256,
                  min_level=2,
                  max_level=6,
                  spatial_scale=[1. / 32., 1. / 16., 1. / 8., 1. / 4.],
-                 has_extra_convs=False):
+                 has_extra_convs=False,
+                 norm_type=False):
```
Comment (qingqing01) [inline PaddleCV/PaddleDetection/ppdet/modeling/backbones/fpn.py]: norm_type=False -> norm_type=None ?

File: PaddleCV/PaddleDetection/ppdet/modeling/backbones/fpn.py [hunk 2]
```diff
@@ -39,35 +40,49 @@ class FPN(object):
         max_level (int): highest level of the backbone feature map to use
         spatial_scale (list): feature map scaling factor
         has_extra_convs (bool): whether has extral convolutions in higher levels
+        norm_type (str): normalization type, 'bn'/'sync_bn'/'affine_channel'
```
Comment (qingqing01) [inline PaddleCV/PaddleDetection/ppdet/modeling/backbones/fpn.py]: (str|None) ?

File: PaddleCV/PaddleDetection/ppdet/modeling/backbones/fpn.py (hunk 1 above)
Comment (Noplz) [reply PaddleCV/PaddleDetection/ppdet/modeling/backbones/fpn.py]: Done

File: PaddleCV/PaddleDetection/ppdet/modeling/backbones/fpn.py (hunk 2 above)
Comment (Noplz) [reply PaddleCV/PaddleDetection/ppdet/modeling/backbones/fpn.py]: Done

**Review by qingqing01 — APPROVED** [review]

File: PaddleCV/PaddleDetection/ppdet/modeling/ops.py [hunk 3]
```diff
@@ -15,15 +15,87 @@
 from numbers import Integral
 
 from paddle import fluid
+from paddle.fluid.param_attr import ParamAttr
+from paddle.fluid.initializer import MSRA
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/ppdet/modeling/ops.py:19]: unnecessary import

**Pull request merged**